addons/core/translations/resources/en-us.yaml

```diff
@@ -586,7 +586,7 @@ role:
         description: Select which scopes you want to add to this role.
         success: Scope selection saved
       keywords-selected:
-        description: Just a heads up, you can't select both add all descendants and add all children. You can only choose one or the other.
+        description: Just a heads up, you can't select both <strong>add all descendants</strong> and <strong>add all children</strong>. You can only choose one or the other.
```

Comment:
None blocking comment: I do understand the <strong> element, I am curious why we are introducing its usage now and never before, do you mind providing a screenshot of this text rendered within PR description?
I do assume this was a suggestion from design?

Reply:
yup this is the specific part that has the bolded text:
![image](https://github.com/user-attachments/assets/6f059cb8-7ab7-4021-8653-6a615860f97d)

It was introduced as feedback and approved by design.